stacks-signer/src/signer.rs

```diff
@@ -1123,6 +1139,7 @@ impl Signer {
     /// Update the DKG for the provided signer info, triggering it if required
     pub fn update_dkg(&mut self, stacks_client: &StacksClient) -> Result<(), ClientError> {
         let reward_cycle = self.reward_cycle;
+        let old_dkg = self.approved_aggregate_public_key;
```

Comment:
Is it possible for `stacks_client` to return `None` because of transient errors?  If so should we check before overwriting the `self` key?  We only seem to be updating the `self.coordinator` key if `stacks_client` returns `Some`